osmoutils/cli_helpers.go

```diff
@@ -47,3 +49,16 @@ func ParseSdkIntFromString(s string, separator string) ([]sdk.Int, error) {
 	}
 	return parsedInts, nil
 }
+
+// ConditionalPanic checks if expectPanic is true, asserts that sut (system under test)
+// panics. If expectPanic is false, asserts that sut does not panic.
+// returns true if sut panics and false it it doesnot
+func ConditionalPanic(t *testing.T, expectPanic bool, sut func()) bool {
+	if expectPanic {
+		require.Panics(t, sut)
+		return true
+	}
+
+	require.NotPanics(t, sut)
+	return false
+}
```

Comment:
```suggestion
func ConditionalPanic(t *testing.T, expectPanic bool, sut func()) {
	if expectPanic {
		require.Panics(t, sut)
	}

	require.NotPanics(t, sut)
}
```

Reply:
@ValarDragon Would this be; cause i'm getting an error without the return
```
func ConditionalPanic(t *testing.T, expectPanic bool, sut func()) {
	if expectPanic {
		require.Panics(t, sut)
                 return 
	}
	require.NotPanics(t, sut)
}
```